contrib/demo/script

```diff
@@ -43,110 +44,65 @@ clear
 pe "kubectl config view --minify | grep server:"
 pe "kubectl config use-context user"
 pe "kubectl config view --minify | grep server:"
-p "kubectl get databases"
-echo "error: the server doesn't have a resource type \"databases\""
+pe "kubectl get databases"
+#echo "error: the server doesn't have a resource type \"databases\""
```

Comment:
There's some leftover commented out stuff here and below.

Reply:
I kept the expected output here on purpose, at least for now.

Reply:
But let me remove them on every call that now fully produces the expected result.